src/components/textarea/styles.ts

```diff
@@ -0,0 +1,79 @@
+import { css } from '@linaria/core'
+import { styled } from '@linaria/react'
+
+export const elTextAreaHasError = css``
+
+export interface TextAreaCSSProperties {
+  '--textarea-max-rows': number
+  '--textarea-min-rows': number
+}
+
+export const ElTextArea = styled.textarea`
```

Comment:
**note:** The code comment below highlights an important decision I made in the implementation of this component. Instead of relying on Linaria's ability to convert `${({ maxRows }) => ...}` into a randomly named CSS variable for us, I've opted to handle it manually in order to provide well-named CSS variables that CSS-only consumers can leverage.

I would have preferred to rely on data attributes on the textarea, but there's currently no way for CSS to use the value of a data attribute. See [attr()](https://developer.mozilla.org/en-US/docs/Web/CSS/attr) for more information.

Reply:
Yep, makes sense - appreciate the comments in here too